package.json

```diff
@@ -51,7 +51,6 @@
     "@babel/preset-env": "^7.24.7",
     "@babel/preset-react": "^7.24.7",
     "@cypress/code-coverage": "^3.12.36",
-    "@cypress/react18": "^2.0.0",
```

Comment:
This must stay and, in cypress/support/component.js, replace 'cypress/react'. However, do we want to do this change in this unrelated commit/PR?

Reply:
You're right, this was removed accidentally, nice catch